static/scss/post.scss

```diff
@@ -287,7 +279,7 @@
     line-height: 29px;
 
     a {
-      color: rgba(0, 0, 0, 0.87);
+      color: $font-black;
     }
```

Comment:
based on [zeplin](https://app.zeplin.io/project/5b181d3bf71408ae4d406b94/screen/5ba4e7e0a4dc98a7608ee6b5) the `.post-title a` element should be the `$navy` color, not black. Probably though we should just put the `.navy` class on the link, instead of having a color set here manually.